compiler/src/dmd/cli.d

```diff
@@ -332,6 +332,15 @@ dmd -cov -unittest myprog.d
             With $(I filename), write module dependencies as text to $(I filename)
             (only imports).`,
         ),
+        Option("dllimport=<value>",
+            "Windows only: select symbols to dllimport (none/defaultLibsOnly/all)",
+            `Which extern(D) global variables to dllimport implicitly if not defined in a root module
```

Comment:
what about extern(C) and extern(C++) symbols?

Reply:
> what about extern(C) and extern(C++) symbols?

The text is a verbatim copy of the LDC command line help. Indeed there seems to be no dependency on the mangling, so I've removed the "extern(D)".
I'm not too happy with `defaultLibsOnly` as it its camel-case style seems different than all other options, but I kept it for compatibility with LDC.

Reply:
I also noticed that the option-value description below referred to LDC-specific options, I removed that. There is currently no change of defaults when using `-shared`.